context/context.go

```diff
@@ -93,16 +92,16 @@ type AMFStatusSubscriptionData struct {
 }
 
 type AppSessionData struct {
-	AppSessionId      string
 	AppSessionContext *models.AppSessionContext
 	// (compN/compN-subCompN/appId-%s) map to PccRule
 	RelatedPccRuleIds    map[string]string
 	PccRuleIdMapToCompId map[string]string
 	// EventSubscription
-	Events   map[models.AfEvent]models.AfNotifMethod
-	EventUri string
+	Events map[models.AfEvent]models.AfNotifMethod
```

Comment:
Same as my other comments, what about doing something like this to keep variables together based on their comments

```diff
diff --git a/context/context.go b/context/context.go
index 7a283fe..57f40f4 100644
--- a/context/context.go
+++ b/context/context.go
@@ -93,16 +92,17 @@ type AMFStatusSubscriptionData struct {
 }

 type AppSessionData struct {
-       AppSessionId      string
        AppSessionContext *models.AppSessionContext
        // (compN/compN-subCompN/appId-%s) map to PccRule
        RelatedPccRuleIds    map[string]string
        PccRuleIdMapToCompId map[string]string
+       // related Session
+       SmPolicyData *UeSmPolicyData
        // EventSubscription
        Events   map[models.AfEvent]models.AfNotifMethod
        EventUri string
-       // related Session
-       SmPolicyData *UeSmPolicyData
+
+       AppSessionId string
 }

 // Create new PCF context
```

Reply:
done